src/lib/plot.typ

```diff
@@ -27,102 +27,165 @@
   return default-plot-style(i)
 }
 
-/// Add an anchor to a plot environment
+/// Create a plot environment. Data to be plotted is given by passing it to the
+/// `plot.add` or other plotting functions. The plot environment supports different
+/// axis styles to draw, see its parameter `axis-style:`.
 ///
-/// - name (string): Anchor name
-/// - position (array): Tuple of x and y values.
-///                     Both values can have the special values "min" and
-///                     "max", which resolve to the axis min/max value.
-///                     Position is in axis space!
-/// - axes (array): Name of the axes to use ("x", "y"), note that both
-///                 axes must exist!
-#let add-anchor(name, position, axes: ("x", "y")) = {
-  ((
-    type: "anchor",
-    name: name,
-    position: position,
-    axes: axes,
-  ),)
-}
-
-/// Create a plot environment
+/// #example(```
+/// import cetz.plot
+/// plot.plot(size: (2,2), x-tick-step: none, y-tick-step: none, {
+///   plot.add(((0,0), (1,1), (2,.5), (4,3)))
+/// })
+/// ```)
+///
+/// To draw elements insides a plot, using the plots coordinate system, use
+/// the `plot.add-annotation(..)` function.
+///
+/// = parameters
+///
+/// = Options
+///
+/// You can use the following options to customize each axis of the plot. You must pass them as named arguments prefixed by the axis name followed by a dash (`-`) they should target. Example: `x-min: 0`, `y-ticks: (..)` or `x2-label: [..]`.
+///
+/// #show-parameter-block("label", ("none", "content"), default: "none", [
+///   The axis' label. If and where the label is drawn depends on the `axis-style`.])
+/// #show-parameter-block("min", ("auto", "float"), default: "auto", [
+///   Axis lower domain value. If this is set greater than than `max`, the axis' direction is swapped])
+/// #show-parameter-block("max", ("auto", "float"), default: "auto", [
+///   Axis upper domain value. If this is set little than than `min`, the axis' direction is swapped])
```

Comment:
Wording: replace "set little than" by "set to a lower value than" or "set to less than"